src/main/java/facade/UserFacade.java

```diff
@@ -39,8 +72,19 @@ public User getUser(String username, String cookie) throws UserNotFoundException
    * @Post User object is in database, user cookie is created.
    * @return User cookie.
    */
-  public String register(User basicUser) {
-    return null;
+  public String register(User basicUser){
+
+      LOGGER.info("UserFacade.register");
+      try {
+          userDao.getUser(basicUser.getUsername());
+          //TODO: throw new UserNotFoundException(); //User was already in the database
+
+      } catch (UserNotFoundException e) {
+          basicUser.setOnline();
```

Comment:
This is probably something that needs to be redesigned in our DAO, but it's probably not a good idea for our "expected path" to be through a catch block.